packages/frontend-v2/utils/course/isEqualCourses.ts

```diff
@@ -1,8 +1,14 @@
-import { ScheduleCourse2 } from "@graduate/common";
+/** Only the details we need to compare courses. */
+type CourseEqualsDetails = {
+  classId: string;
+  subject: string;
+} & {
+  [key: string]: any;
+};
 
```

Comment:
Just as a clarification, is there a benefit to specifying string keys for extra properties versus just doing `& any`?

Reply:
Yeap! With & any the following would be fine by TS
```
isEqualCourses({ yo: "hi" }, { yo: 2 });
```

Because the & any essentially makes the whole type any. But the way it currently is makes it so that at least classId and subject are required. But other keys could also exist.